front/lib/api/assistant/conversation.ts

```diff
@@ -269,7 +279,7 @@ export async function getConversation(
   auth: Authenticator,
   conversationId: string,
   includeDeleted?: boolean
-): Promise<ConversationType | null> {
+): Promise<Result<ConversationType, ConversationPermissionError>> {
```

Comment:
Note: did *not* do the restriction check for the `getUserConversations` function a few lines above here (above `getConversation`)

Reason is it would become too heavy. Not checking is fine IMO since 1. already filtered by user being a participant, 2. not leaking conversation content. If conversation content on a conversation that became private were to be requested, then the restriction *would* apply.